lib/driver.js

```diff
@@ -239,6 +243,43 @@ class XCUITestDriver extends BaseDriver {
     }
   }
 
+  async getWDAAttachmentsRoots() {
```

Comment:
Since neither of these functions need to use any state on the driver, it would be much better to move it all to the `lib/utils.js`.